components/01-atoms/TestenetBanner.tsx

```diff
@@ -0,0 +1,7 @@
+export const TestenetBanner = () => {
+  return (
+    <div className="bg-yellow-100 w-full p-2 flex items-center justify-center">
+      You are viewing the Nameful app on Sepolia testnet.
```

Comment:
I'd rather see the chain name coming from Wagmi since we will soon have new L2s integrated and we want this to be easily variable.